docs/content/ingestion/tasks.md

```diff
@@ -114,11 +114,13 @@ The tuningConfig is optional and default parameters will be used if no tuningCon
 |type|The task type, this should always be "index".|none|yes|
 |targetPartitionSize|Used in sharding. Determines how many rows are in each segment.|5000000|no|
 |maxRowsInMemory|Used in determining when intermediate persists to disk should occur.|75000|no|
+|maxTotalRows|Total number of rows in segments waiting for being published. Used in determining when intermediate publish should occur.|150000|no|
```

Comment:
Can you add maybe another section to the docs about the task's publishing behavior? I think it'd be useful to have one place where the user can see what publishing modes are supported (early vs. no early) and what configuration parameters affect the publishing behavior